tap_nice_incontact/schemas/qm_workflows.json

```diff
@@ -0,0 +1,133 @@
+{
+  "type": ["null", "object"],
+  "additionalProperties": false,
+  "properties": {
+    "tenantName": {
+      "type": ["null", "string"]
+    },
+    "agentId": {
+      "type": ["null", "string"]
+    },
+    "type": {
+      "type": ["null", "string"]
+    },
+    "evaluationFlowTrigger": {
+      "type": ["null", "string"]
+    },
+    "workflowStatus": {
+      "type": ["null", "string"]
+    },
+    "autoAcknowledged": {
+      "type": ["null", "string"]
+    },
+    "workflowState": {
+      "type": ["null", "string"]
+    },
+    "workflowId": {
+      "type": ["null", "string"]
+    },
+    "planName": {
+      "type": ["null", "string"]
+    },
+    "formName": {
+      "type": ["null", "string"]
+    },
+    "formVersion": {
+      "type": ["null", "string"]
+    },
+    "startDate": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "submissionDate": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "lastUpdated": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "scorePercentage": {
+      "type": ["null", "string"]
+    },
+    "score": {
+      "type": ["null", "string"]
+    },
+    "maximumPossibleScore": {
+      "type": ["null", "string"]
+    },
+    "rank": {
+      "type": ["null", "string"]
+    },
+    "failedCriticalQuestion": {
+      "type": ["null", "string"]
+    },
+    "userReviewComments": {
+      "type": ["null", "string"]
+    },
+    "disputeResolverComments": {
+      "type": ["null", "string"]
+    },
```

Comment:
potentially array of objects?

Reply:
I looked through many records but couldn't see an example of what the shape of the data in these fields look like (they are all empty).

I'm going to leave this as a string for now, we can update this if we encounter any issues when using this field.
<img width="741" alt="Screen Shot 2023-01-04 at 5 54 40 PM" src="https://user-images.githubusercontent.com/1455838/210683930-8468a517-a33b-4ff3-8e54-36939093ebb1.png">